### PR TITLE
Zombie Revert (now it actually works)

### PR DESCRIPTION
**Pull request by Gandalf2k15**
This fixes zombies! Finally! We can have non-laggy ass shitty server crashing zombies! However it comes at a price, they are slightly less advanced as they are TG's OLD zombies. I have tested this multiple times on my test server and it works like a charm. I invited people and lag was non existent. Finally we can have infestations with zombies! Due to it being so very simple I don't see how it could lag anymore.

### CODE REVIEW

**Comment by Zadeon** [comment]
:+1:

**Comment by Gandalf2k15** [comment]
Oh fuck, It needs to be compiled.

**Comment by yobob591** [comment]
🎃👻🎃👻🎃👻👻👻🎃👻 spooky shit spooky sHit🎃 thats ✔ some spooky🎃🎃shit right🎃🎃th 🎃 ere🎃🎃🎃 right✔there ✔✔if i do ƽaү so my selｆ 💯 i say so 💯 thats what im talking about right there right there (chorus: ʳᶦᵍʰᵗ ᵗʰᵉʳᵉ) mMMMMᎷМ💯 🎃🎃 🎃НO0ОଠＯOOＯOОଠଠOoooᵒᵒᵒᵒᵒᵒᵒᵒᵒ🎃 🎃 🎃 🎃 💯 🎃 👻👻 👻 🎃🎃spooky shit

**Comment by calibraptor** [comment]
+1 if you can fix the checks failing.

**Comment by Gandalf2k15** [comment]
Fixed it, pushing in a sec.

**Comment by Gandalf2k15** [comment]
Fixed, ready to merge.

**Comment by Crystalwarrior** [comment]
Hold on, what exactly does this do? ARe you sure this is performance fix ONLY or are you removing any features?

**Comment by Crystalwarrior** [comment]
Looking at code, you definitely changed the balancing of zombies as well.

File: code/game/gamemodes/zombie/zombie.dm [hunk 1]
```diff
@@ -1,3 +1,4 @@
+/*
```
Comment (Crystalwarrior) [inline code/game/gamemodes/zombie/zombie.dm:1]: ...Why did you comment out the entire gamemode? This entire code allows zombies to be mulligan.

File: code/modules/mob/living/simple_animal/hostile/zombie.dm [hunk 2]
```diff
@@ -1,184 +1,153 @@
...
+	speak_emote = list("groans")
+	emote_see = list("groans")
 	a_intent = "harm"
 	maxHealth = 120
 	health = 120
 	speed = 2
-	move_to_delay = 5
 	harm_intent_damage = 8
-	melee_damage_lower = 9
+	melee_damage_lower = 20
```
Comment (Crystalwarrior) [inline code/modules/mob/living/simple_animal/hostile/zombie.dm:16]: Why did you up melee_damage_lower? Now zombies will consistently dish out 20 damage per attack. Is that what you want?

**Comment by Crystalwarrior** [comment]
Okay what the fuck, this PR isn't a fix, it's a straight up revert back to the fucking /tg/ zombies.
DO NOT MERGE.

File: code/modules/mob/living/simple_animal/hostile/zombie.dm [hunk 3]
```diff
@@ -1,184 +1,153 @@
...
 	maxbodytemp = 350
 	unsuitable_atmos_damage = 10
-	environment_smash = 1 //Can smash tables and lockers, etc.
-	can_force_doors = 1 //Can force doors open like a champ
+	environment_smash = 1
 	robust_searching = 1
-	stat_attack = 1 //Can attack unconscious players
-	gold_core_spawnable = 0 //No.
+	stat_attack = 2
+	gold_core_spawnable = 1
```
Comment (Crystalwarrior) [inline code/modules/mob/living/simple_animal/hostile/zombie.dm:27]: yeah great job now old /tg/ zombies will be able to ruin rounds with gold cores once more

File: code/modules/mob/living/simple_animal/hostile/zombie.dm [hunk 4]
```diff
@@ -1,184 +1,153 @@
...
+	H.stat = DEAD
+	H.butcher_results = list(/obj/item/weapon/reagent_containers/food/snacks/meat/slab/human/mutant/zombie = 3) //So now you can carve them up when you kill them. Maybe not a good idea for the human versions.
+	H.loc = Z
+	Z.stored_corpse = H
+	for(var/mob/living/simple_animal/hostile/zombie/holder/D in H) //Dont want to revive them twice
+		qdel(D)
+	visible_message("<span class='danger'>[Z] staggers to their feet!</span>")
+	Z << "<span class='userdanger'>You are now a zombie! Follow your creators lead!</span>"
+
+/mob/living/simple_animal/hostile/zombie/holder
```
Comment (Crystalwarrior) [inline code/modules/mob/living/simple_animal/hostile/zombie.dm:133]: What even is this?

**Comment by Gandalf2k15** [comment]
Ignore what crystal is saying. It works well with this code.

**Comment by Gandalf2k15** [comment]
What this does is removes old broken content in favour of older working content. Crystals attempt to fix em failed. I forgot to remove some things and I will get to this later. Once done this is good to go. I'll nerf the damage too. But damn crystal you don't know how fucked you made the old zombies. Why keep them. The code is ugly as fuck. It's unsalvageable.

**Comment by Gandalf2k15** [comment]
Crystal is fixing em

**Pull request closed without merging**